client/my-sites/plans/current-plan/controller.jsx

```diff
@@ -31,6 +31,11 @@ export function currentPlan( context, next ) {
 		return null;
 	}
 
-	context.primary = <CurrentPlan path={ context.path } />;
+	context.primary = (
+		<CurrentPlan
+			path={ context.path }
+			showThankYou={ context.query.hasOwnProperty( 'thank-you' ) }
```

Comment:
Open to suggestions on this param.

Reply:
I like the param name, but should we map the prop name (`showThankYou`) to what it actually shows? It shows a thank you header, so do we want to make it `showThankYouHeader` instead?

Or another way would be to totally rephrase "Thank you" to "Success" everywhere?

Reply:
I think we're good as-is 🙂